paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -94,6 +95,7 @@ internal class CustomerSheetViewModel @Inject constructor(
     private var paymentLauncher: PaymentLauncher? = null
 
     private var unconfirmedPaymentMethod: PaymentMethod? = null
+    private var bankAccountResult: CollectBankAccountResultInternal? = null
```

Comment:
Should we model this as part of the view state? Then we can also add a computed property for `viewState.canDismissSheet` or something like that.